parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -35,6 +35,11 @@ removeContainerd() {
     fi
 }
 
+disableTimeSyncd() {
+    systemctl_stop 20 5 10 systemd-timesyncd || exit $1
```

Comment:
why not just 
```suggestion
    systemctl_stop 20 5 10 systemd-timesyncd || exit $ERR_SYSTEMCTL_STOP_FAIL
```
to stay consistent with the rest of the file? We're not passing in exit codes as args in other CSE functions

Reply:
Good question! I'm just sort of desparate to save cloud-init bytes here is the reason.

Reply:
in that case maybe we can change `systemctl_stop` to take more than one arg and stop both in one call

Reply:
I think that's a good suggestion for a follow-up PR, how about my latest change?

Reply:
I think it's okay but I really don't like that we have to lower code quality/readability to save space :(

Reply:
:(